.github/workflows/new-milestone-and-issue-in-other-repo.yml

```diff
@@ -0,0 +1,36 @@
+#############################################################################
+# GitHub Action to Create New Milestone and issue in Other Repositories.
+#
+#############################################################################
+name: "R: Close/Create Milestone, Upload '-all' jar, Create issues"
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Target Version without (-SNAPSHOT)'
+        required: true
+jobs:
+  creation:
+    name: Creation-New Milestone/Issue" for ${{ github.event.inputs.version }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the latest code
+        uses: actions/checkout@v2
+      - name: Close/Create Milestone
+        run: |
+          ./.ci/close-create-milestone.sh ${{ github.event.inputs.version }}
+      - name: Upload '-all' jar
+        run: |
+          UPLOAD_LINK=https://uploads.github.com/repos/checkstyle/checkstyle/releases
```

Comment:
Move it back to shell script and name script  as upload-all-jar.sh

Reply:
Changes Done